xml/libvirt_overview.xml

```diff
@@ -19,65 +19,393 @@
-  command:
+  The modular daemon deployment is useful in scenarios where minimal libvirt
+  support is needed. For example if virtual machine storage and networking is
+  not provided by libvirt, the various libvirt-daemon-driver-storage and
+  libvirt-daemon-driver-network packages are not required. Kubernetes is an
+  example of an extreme case, where it handles all networking, storage, cgroups
+  and namespace integration, etc. Only the libvirt-daemon-driver-qemu package,
+  providing virtqemud, needs to be installed. Modular daemons allow configuring
+  a custom libvirt installation containing only the components required for
+  the use-case.
  </para>
- <screen>&prompt.sudo;systemctl enable libvirtd</screen>
+ 
+ <sect1 xml:id="libvirt-monolithic-daemon">
+  <title>Starting and stopping the monolithic daemon</title>
+
+  <para>
+   The monolithic daemon is known as libvirtd and is configured via
+   /etc/libvirt/libvirtd.conf. libvirtd is managed with several systemd unit
+   files:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     <emphasis>libvirtd.service</emphasis> - The main systemd unit file for
+     launching libvirtd. It is recommended to configure libvirtd.service to
+     start on boot if VMs are also configured to start on host boot.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd.socket</emphasis> - The unit file corresponding to the
+     main read-write UNIX socket /var/run/libvirt/libvirt-sock. This socket is
+     recommended to be started on boot by default.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd-ro.socket</emphasis> - The unit file corresponding to
+     the main read-only UNIX socket /var/run/libvirt/libvirt-sock-ro. This
+     socket is recommended to be started on boot by default.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd-admin.socket</emphasis> - The unit file corresponding
+     to the administrative UNIX socket /var/run/libvirt/libvirt-admin-sock.
+     This socket is recommended to be started on boot by default.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd-tcp.socket</emphasis> - The unit file corresponding
+     to the TCP 16509 port for non-TLS remote access. This socket should not
+     be configured to start on boot until the administrator has configured a
+     suitable authentication mechanism.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis>libvirtd-tls.socket</emphasis> - The unit file corresponding
+     to the TCP 16509 port for TLS remote access. This socket should not be
+     configured to start on boot until the administrator has deployed x509
+     certificates and optionally configured a suitable authentication mechanism.
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <para>
+   When systemd socket activation is used a number of configuration settings
```

Comment:
do we explain elsewhere what systemd socket activation means?

Reply:
Isn't it a well known systemd-ism?

Reply:
right we can probably rely on systemd info, although there isn't that much:

https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-systemd.html